### PR TITLE
ifeellike

### PR DESCRIPTION
**Pull request by caltrops6490**
xddd gg no re

### CODE REVIEW

File: binstar_client/commands/move.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+from binstar_client.utils import get_server_api, parse_specs
+
+
+logger = logging.getLogger('binstar.move')
+
+
+def main(args):
+    aserver_api = get_server_api(args.token, args.site)
+
+    spec = args.spec
```
Comment (caltrops6490) [inline binstar_client/commands/move.py:21]: like this?

File: binstar_client/commands/move.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        help='Label to move packages from',
+        default='main',
+    )
+    _to.add_argument(
+        '--to-label',
+        help='Label to move packages to',
+        default='main',
+    )
+
+    parser.set_defaults(main=main)
```
Comment (caltrops6490) [inline binstar_client/commands/move.py:105]: dear god

File: .travis.yml [hunk 3]
```diff
@@ -4,36 +4,96 @@ language: python
...
+      $SUDO rm /usr/bin/lsb_release;
+      $SUDO cp -r /home/travis/miniconda/envs/test/lib/python$PYTHON_VERSION /usr/lib/python$PYTHON_VERSION;
+    fi;
+  - pip install --user $PIP_TEST_DEPENDENCIES
+
+  # Install this package
+  - pip install --user -e .
 
 script:
-  - python binstar_client/tests/runtests.py
```
Comment (caltrops6490) [inline .travis.yml:35]: d

**Comment by caltrops6490** [comment]
so is that good ?

**Review by ANUPRTATUSKAR — COMMENTED** [review]
printf("Hello baby")

**Pull request closed without merging**